.github/workflows/tests.yml

```
@@ -44,31 +44,31 @@ jobs:
 
-    - name: Run Flask server in background
-      run: |
-        poetry run flask run &
-        echo $! > flask_pid.txt
-        sleep 3
+    # - name: Run Cypress tests
+    #   working-directory: tests/e2e_tests
+    #   run: npx cypress run --spec "cypress/e2e/**/*"
 
-# Run Cypress tests specifying the folder with your test specs
+<<<<<<< HEAD
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 60: could not find expected ':'

(syntax-check)